package.json

```diff
@@ -61,52 +61,50 @@
     "@types/chokidar": "^1.7.5",
     "@types/minimist": "^1.2.0",
     "@types/pretty-ms": "^3.0.0",
-    "acorn": "^5.5.3",
+    "acorn": "^5.7.1",
     "acorn-dynamic-import": "^3.0.0",
     "acorn-import-meta": "^0.2.1",
     "ansi-escapes": "^3.1.0",
     "buble": "^0.19.3",
     "chalk": "^2.4.1",
-    "codecov.io": "^0.1.6",
     "console-group": "^0.3.1",
     "date-time": "^2.1.0",
     "eslint": "^4.19.1",
     "eslint-plugin-import": "^2.12.0",
     "execa": "^0.10.0",
     "fixturify": "^0.3.4",
-    "hash.js": "^1.1.3",
+    "hash.js": "^1.1.4",
     "husky": "^0.14.3",
     "immutable": "^3.8.2",
     "is-reference": "^1.1.0",
     "istanbul": "^0.4.3",
-    "lint-staged": "^7.1.1",
+    "lint-staged": "^7.2.0",
     "locate-character": "^2.0.5",
     "magic-string": "^0.25.0",
     "minimist": "^1.2.0",
     "mocha": "^5.2.0",
-    "prettier": "^1.12.1",
-    "pretty-ms": "^3.1.0",
+    "prettier": "^1.13.5",
+    "pretty-ms": "^3.2.0",
     "remap-istanbul": "^0.11.1",
     "require-relative": "^0.8.7",
-    "rollup": "^0.60.7",
+    "rollup": "latest",
     "rollup-plugin-buble": "^0.19.2",
     "rollup-plugin-commonjs": "^9.1.3",
     "rollup-plugin-json": "^3.0.0",
     "rollup-plugin-node-resolve": "^3.3.0",
     "rollup-plugin-replace": "^2.0.0",
-    "rollup-plugin-string": "^2.0.0",
+    "rollup-plugin-string": "^2.0.2",
     "rollup-plugin-typescript": "^0.8.1",
-    "rollup-pluginutils": "^2.2.0",
-    "rollup-watch": "^4.3.1",
+    "rollup-pluginutils": "^2.3.0",
     "sander": "^0.6.0",
```

Comment:
Not sure the hypothesis stands on the delay?

Reply:
What I observed is that the watcher would trigger noticeably before sander.writeFileSync would return. Might also be related to an update of a transitive dependency.